main.go

```diff
@@ -142,6 +147,7 @@ func main() {
 		RHOBSSecretName:              fmt.Sprintf("%v-prom-remote-write", addonName),
 		RHOBSEndpoint:                envVars[rhobsEndpointEnvVarName],
 		RHSSOTokenEndpoint:           envVars[rhssoTokenEndpointEnvVarName],
+		CRDExist:                     checkIfCRDExist(controllers.GetCRDNames()),
```

Comment:
`controller.GetCRDNames` does not make sense here. 
As a reader I would like to be able to understand which CRDs are we looking for without the need to jump into another file.